README.md

```diff
@@ -24,10 +24,11 @@ The [master](https://github.com/butterproject/butter-desktop) branch which conta
 
 #### Quickstart:
 
+
 1. `npm install -g grunt-cli bower`
-1. `npm install`
-1. `grunt build`
-1. `grunt start`
+2. `npm start`
```

Comment:
@xaiki The the part that I have edited out was the quickstart part, which is only supposed to show what you need to quickstart not all the commands that are you are running with npm start. The more informative version of the commands is found right under the Quickstart commands. I didn't bother to explain because I thought you would understand already, sigh.

Anyways, if you still think that the quickstart command should show all the commands that are being run by npm start, I'll push another change.